### DISCUSSION TITLE
v3: Minor Improvements Feedback

### DISCUSSION THREAD

**Discussion by ramonmalcolm10** [livewire/livewire#5804] [3. Feature ideas]
I went through the documentation and didn’t saw any recommended way on how to achieve the following for full page components:
- Add specific styles per component
- Not able to set meta description, a attribute similar to the Title attribute would be nice or guide on how to achieve same
- How to integrate with google analytics
- Would be good if we can add a head in these components that would update the page head

One other issue identified on the quick start, it stated that the default layout is found at resources/views/components/layout.blade.php which the correct location is resources/views/components/layouts/app.blade.php

**Comment by StaffCollab** [comment]
Hi @ramonmalcolm10 

Since Livewire components are blade components, you can use `@stack('scripts')` and `@stack('styles')` for them.
https://laravel.com/docs/10.x/blade#stacks
